src/bodywork/cli/cli.py

```diff
@@ -474,9 +474,12 @@ def service(args: Namespace) -> None:
     elif command == "delete":
         load_kubernetes_config()
         delete_service_deployment_in_namespace(namespace, name)
+    elif name is not None:
```

Comment:
Could you get rid of this `elif` block by specifying `None` as a default value in,

```python
service_cmd_parser.add_argument(
     "--namespace",
     "--ns",
     required=False,
     type=str,
     help="Kubernetes namespace to operate in.",
 )
```

?

Reply:
I think `None` will be the default if no default is specified but we want to make sure we filter on empty strings as well which I believe would occur if you did `--name` but forgot to specify a value. 

However this `if` statement should be just `elif name` so i'll fix that. Also I can remove some repetition in the whole block so I am going to do a little bit of refactoring too.

Reply:
You're correct - `None` is the default (just checked). However, you'll get an error if you forgot to specify a value for `--name`, so no need to filter on empty strings unless someone explicitly passes `""`.

Reply:
--name is not a required parameter i.e. `required=false` therefore an error won't be thrown if it's not specified. Either way it's best to leave the code defensive and cover the explicit `""` scenario

Reply:
If you don't specify `--name` does it equal `""` or `None`?

Reply:
This is obsolete now, I forgot to mention in the comments (I think I just did on Slack) that I implemented the feature so that we can search for an individual service so now we use --name and it is the same method that is called with optional parameters so this is now just an `else`.

Reply:
💥